airflow/models/baseoperator.py

```diff
@@ -1057,6 +1070,17 @@ def name(self) -> str:
         :return: link name
         """
 
+    @property
+    def operator_name(self) -> Optional[str]:
+        """
+        This property will be used by Airflow Plugins to find the class to which you want
+        to assign this Operator Link
+
+        :return: Name of Operator used by task for which
```

Comment:
Probably the full class path here?

Reply:
Yeah, I think full class path, or perhaps the class itself?

Reply:
Updated the PR

Reply:
Oh, docs still need updating though.

Reply:
Does this need to be a property? Could we instead just do

```
    operators = None  # type: Optional[List[BaseOperator]]
    """
    This property will be used by Airflow Plugins to find the Operators to which you want
    to assign this Operator Link
    """
```

(It's slightly odd, but to document a property the string goes after it.

Reply:
Updated